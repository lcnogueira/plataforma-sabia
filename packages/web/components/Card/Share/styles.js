import styled, { css, keyframes } from 'styled-components';
import { AiOutlineShareAlt } from 'react-icons/ai';

const zoom = keyframes`
	0%, 100% {
		transform: scale(1);
	}
	50% {
		transform: scale(1.1);
	}
`;

export const Container = styled.button`
	${({ theme: { sizes } }) => css`
		display: flex;
		align-items: center;
		justify-content: space-between;
		padding: 0.5rem;
		gap: 0.6rem;
		cursor: pointer;
		background: transparent;
		border: none;

		svg {
			width: ${sizes.defaultIcon}rem;
			height: ${sizes.defaultIcon}rem;

			&:hover {
				animation: ${zoom} 1.5s;
			}
		}
	`}
`;

export const Icon = styled(AiOutlineShareAlt).attrs(({ theme: { colors } }) => ({
	color: colors.lightGray2,
}))``;
