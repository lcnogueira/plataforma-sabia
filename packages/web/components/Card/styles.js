import styled, { css } from 'styled-components';

export const CardContainer = styled.div`
	${({ theme: { colors, metrics } }) => css`
		border-radius: ${metrics.baseRadius}rem;

		display: flex;
		flex-direction: column;
		background-color: ${colors.white};
		transition: all 0.3s ease 0s;
		padding: 0.8rem;

		:hover {
			transform: translateY(-0.7rem);
		}
	`}
`;

export const ImageContainer = styled.div`
	${({ theme: { metrics } }) => css`
		border-radius: ${metrics.baseRadius}rem;
		width: 100%;

		overflow: hidden;
		position: relative;
	`}
`;

export const Badge = styled.div`
	position: absolute;
	left: 0;
	padding: 0.4rem 0.8rem;
	font-size: 1.4rem;
	line-height: 2.4rem;
	border-top-right-radius: ${({ theme }) => theme.metrics.baseRadius}rem;
	border-bottom-right-radius: ${({ theme }) => theme.metrics.baseRadius}rem;

	${(props) =>
		props.bottom &&
		css`
			bottom: 2rem;
			background-color: ${({ theme }) => theme.colors.white};
			color: ${({ theme }) => theme.colors.silver};
		`}

	${(props) =>
		props.top &&
		css`
			top: 2rem;
			background-color: ${({ theme }) => theme.colors.white};
			color: ${({ theme }) => theme.colors.black};
		`}
`;

export const Content = styled.div`
	display: flex;
	flex-direction: column;
	margin-top: 1.6rem;
	font-weight: 400;
`;

export const MainTitle = styled.h3`
	font-family: 'Montserrat';
	font-weight: 500;
	font-size: 1.6rem;
	margin-bottom: 3rem;
	line-height: 2.4rem;
`;

export const TextContainer = styled.div`
	display: flex;
	align-items: center;
	justify-content: space-between;
	font-size: 1.2rem;
	padding-top: 0.8rem;
	border-top: 0.1rem solid ${({ theme }) => theme.colors.border};
`;

export const InstitutionText = styled.span`
	${({ theme: { colors } }) => css`
		display: flex;
		align-items: center;
		justify-content: center;
		font-weight: 500;
		color: ${colors.lightGray2};
		text-transform: uppercase;

		> svg {
			path {
				stroke: ${colors.lightGray2};
				stroke-width: 1.5;
			}
			margin-right: 0.8rem;
		}
	`}
`;

export const CalendarText = styled.div`
	display: flex;
	align-items: center;

	span {
		color: ${({ theme }) => theme.colors.secondary};
		margin-left: 0.5rem;
		display: block;
	}

	svg {
		width: ${({ theme }) => theme.sizes.smallIcon}rem;
		height: ${({ theme }) => theme.sizes.smallIcon}rem;
	}
`;

export const LikesWrapper = styled.div`
	margin-bottom: 1rem;
	> div {
		padding: 0;
	}
`;

export const Price = styled.h2`
	${({ theme: { colors } }) => css`
		font-size: 2.8rem;
		line-height: 3.3rem;
		color: ${colors.secondary};
		margin-bottom: 1.2rem;
	`}
`;
