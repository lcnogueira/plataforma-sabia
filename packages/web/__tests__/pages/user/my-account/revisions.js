import React from 'react';
import { render, screen } from 'test-utils';
import Revisions from '../../../../pages/user/my-account/revisions';

import * as useAuth from '../../../../hooks/useAuth';
import { ROLES } from '../../../../utils/enums/api.enum';
import { STATUS } from '../../../../utils/enums/technology.enums';
import { theme } from '../../../../styles';

const technologies = [
	{
		id: 1,
		title: 'Fake technology',
		updated_at: '2020-10-19T14:27:58.543Z',
		status: STATUS.IN_REVIEW,
	},
	{
		id: 2,
		title: 'Fake technology',
		updated_at: '2020-10-19T14:27:58.543Z',
		status: STATUS.REQUESTED_CHANGES,
	},
	{
		id: 3,
		title: 'Fake technology',
		updated_at: '2020-10-19T14:27:58.543Z',
		status: STATUS.CHANGES_MADE,
	},
];

test('it renders Revisions page correctly', () => {
	jest.spyOn(useAuth, 'default').mockReturnValue({
		user: {
			email: 'test@test.com',
			role: {
				role: ROLES.REVIEWER,
			},
		},
	});

	const { container } = render(
		<Revisions
			technologies={technologies}
			currentPage={1}
			totalPages={1}
			totalItems={3}
			itemsPerPage={5}
		/>,
	);

	const inReviewEl = screen.getByText('Aguardando análise');
	const requestedChangesEl = screen.getByText('Aguardando correção');
	const changesMadeEl = screen.getByText('Correção efetuada');

	expect(container).toMatchSnapshot();

	expect(inReviewEl).toBeInTheDocument();
	expect(inReviewEl).toHaveStyle({ color: theme.colors.lightGray2 });

	expect(requestedChangesEl).toBeInTheDocument();
	expect(requestedChangesEl).toHaveStyle({ color: theme.colors.primary });

	expect(changesMadeEl).toBeInTheDocument();
	expect(changesMadeEl).toHaveStyle({ color: theme.colors.secondary });
});
