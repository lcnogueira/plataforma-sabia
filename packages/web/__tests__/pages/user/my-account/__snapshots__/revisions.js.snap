// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders Revisions page correctly 1`] = `
.c7 {
  font-size: 2.8rem;
  font-weight: 500;
  text-align: center;
  color: hsl(166,100%,34%);
  text-align: left;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  font-size: 1.6rem;
  background: none;
  border: none;
  color: hsla(0,79%,63%);
  padding-left: 2rem;
}

.c5:hover {
  color: hsla(0,58%,57%);
}

.c5 svg {
  fill: hsla(0,79%,63%);
  stroke: hsla(0,79%,63%);
  width: 2rem;
  height: 2rem;
  margin-right: 1rem;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-size: 1.4rem;
  font-weight: 500;
  margin-bottom: 2rem;
  color: hsla(0,0%,52%);
  padding-left: 2rem;
}

.c4 svg {
  margin-right: 1rem;
}

.c4:hover {
  color: hsla(146,64%,36%);
}

.c1 {
  min-width: 30rem;
}

.c2 {
  display: block;
  padding-left: 2rem;
  font-size: 1.6rem;
  color: hsl(166,100%,34%);
}

.c2 span {
  font-weight: bold;
}

.c3 {
  font-family: 'Montserrat';
  font-size: 1.4rem;
  margin: 2rem 0;
  padding: 0.5rem 0;
  text-align: center;
  text-transform: uppercase;
  color: hsla(0,0%,40%);
  background-color: hsla(0,0%,95%);
}

.c9 {
  width: 100%;
}

.c10 {
  margin: 0;
  padding: 0;
}

.c16 {
  list-style: none;
  border-radius: 0.2rem;
  text-align: center;
  background-color: hsla(0,0%,100%);
  font-size: 1.4rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-top: 0.1rem solid hsla(0,0%,93%);
  border-bottom: 0.1rem solid hsla(0,0%,93%);
  color: hsl(166,100%,34%);
  font-weight: bold;
  font-size: 1.6rem;
}

.c16 a {
  font-weight: 400;
  font-size: 1.4rem;
}

.c18 {
  list-style: none;
  border-radius: 0.2rem;
  text-align: center;
  background-color: hsla(0,0%,100%);
  font-size: 1.4rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: hsla(0,0%,100%);
}

.c18 a {
  font-weight: 400;
  font-size: 1.4rem;
}

.c20 {
  list-style: none;
  border-radius: 0.2rem;
  text-align: center;
  background-color: hsla(0,0%,100%);
  font-size: 1.4rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: hsla(0,0%,95%);
}

.c20 a {
  font-weight: 400;
  font-size: 1.4rem;
}

.c17 {
  padding: 2rem 1rem;
  word-break: break-word;
}

.c13 {
  margin-right: 0.5rem;
  width: 2.2rem;
  height: 2.2rem;
}

.c14 {
  margin-left: 0.5rem;
  width: 2.2rem;
  height: 2.2rem;
}

.c12 {
  width: 2.2rem;
  height: 2.2rem;
}

.c15 {
  width: 2.2rem;
  height: 2.2rem;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c11 button {
  color: hsla(0,0%,40%);
  background: none;
  border: none;
}

.c11 button:disabled {
  cursor: not-allowed;
  opacity: 0.5;
}

.c11 .sort-by {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c11 .sort-by .sort-order {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c11 .sort-by > div {
  margin-right: 1.2rem;
}

.c11 .sort-by > div > button:last-of-type {
  margin-left: 1.2rem;
}

.c11 .pagination {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-left: auto;
}

.c11 .pagination > span {
  font-size: 1.4rem;
}

.c11 .sort-by > div,
.c11 .pagination {
  margin-bottom: 1.4rem;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0 auto;
  background-color: hsla(0,0%,98%);
  padding: 3rem 4rem 6rem;
}

.c0 > section:first-child {
  margin-right: 4rem;
}

.c6 {
  width: 100%;
}

.c8 {
  min-height: 80vh;
  background-color: hsla(0,0%,100%);
  padding: 2rem;
}

.c19 {
  display: inline-block;
  position: relative;
  line-height: 2.4rem;
  font-weight: 500;
  padding: 0.2rem 0.8rem;
  max-width: -webkit-fit-content;
  max-width: -moz-fit-content;
  max-width: fit-content;
  text-align: center;
  color: hsla(0,0%,52%);
}

.c19::before {
  content: '';
  display: block;
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  border-radius: 1.45rem;
  opacity: 0.1;
}

.c19::before {
  background: hsla(0,0%,52%);
}

.c21 {
  display: inline-block;
  position: relative;
  line-height: 2.4rem;
  font-weight: 500;
  padding: 0.2rem 0.8rem;
  max-width: -webkit-fit-content;
  max-width: -moz-fit-content;
  max-width: fit-content;
  text-align: center;
  color: hsla(29,94%,62%);
}

.c21::before {
  content: '';
  display: block;
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  border-radius: 1.45rem;
  opacity: 0.1;
}

.c21::before {
  background: hsla(29,94%,62%);
}

.c22 {
  display: inline-block;
  position: relative;
  line-height: 2.4rem;
  font-weight: 500;
  padding: 0.2rem 0.8rem;
  max-width: -webkit-fit-content;
  max-width: -moz-fit-content;
  max-width: fit-content;
  text-align: center;
  color: hsl(166,100%,34%);
}

.c22::before {
  content: '';
  display: block;
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  border-radius: 1.45rem;
  opacity: 0.1;
}

.c22::before {
  background: hsl(166,100%,34%);
}

@media (max-width:768px) {
  .c7 {
    font-size: 2.4rem;
  }
}

@media screen and (max-width:1170px) {
  .c10 {
    display: grid;
    grid-template-columns: 1fr;
    grid-gap: 2rem;
  }
}

@media screen and (max-width:1170px) {
  .c16 {
    display: none;
  }
}

@media screen and (min-width:1170px) {
  .c16 {
    display: grid;
    grid-template-columns: repeat(3,minmax(0,1fr));
  }
}

@media screen and (max-width:1170px) {
  .c16 {
    border: 0.1rem solid hsla(0,0%,93%);
    padding: 1rem;
  }
}

@media screen and (min-width:1170px) {
  .c18 {
    display: grid;
    grid-template-columns: repeat(3,minmax(0,1fr));
  }
}

@media screen and (max-width:1170px) {
  .c18 {
    border: 0.1rem solid hsla(0,0%,93%);
    padding: 1rem;
  }
}

@media screen and (min-width:1170px) {
  .c20 {
    display: grid;
    grid-template-columns: repeat(3,minmax(0,1fr));
  }
}

@media screen and (max-width:1170px) {
  .c20 {
    border: 0.1rem solid hsla(0,0%,93%);
    padding: 1rem;
  }
}

@media screen and (max-width:1170px) {
  .c17 {
    display: grid;
    grid-template-columns: minmax(9rem,40%) 1fr;
    text-align: left;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }

  .c17::before {
    content: attr(data-name);
    color: hsl(166,100%,34%);
    font-weight: bold;
  }
}

@media screen and (max-width:950px) {
  .c0 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }

  .c0 > section:first-child {
    margin-bottom: 1rem;
  }
}

<div>
  <div
    class="c0"
  >
    <section
      class="c1"
    >
      <div
        class="c2"
      >
        profile:welcomeUser
      </div>
      <h3
        class="c3"
      >
        profile:userArea
      </h3>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M20 21v-2a4 4 0 00-4-4H8a4 4 0 00-4 4v2M12 11a4 4 0 100-8 4 4 0 000 8z"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:myProfile
      </a>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M22 6l-10 7L2 6"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:messages
      </a>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="20"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M20.84 2.61a5.5 5.5 0 00-7.78 0L12 3.67l-1.06-1.06a5.501 5.501 0 00-7.78 7.78l1.06 1.06L12 19.23l7.78-7.78 1.06-1.06a5.5 5.5 0 000-7.78v0z"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:favoriteTechnologies
      </a>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M6 2L3 6v14a2 2 0 002 2h14a2 2 0 002-2V6l-3-4H6zM3 6h18M16 10a4 4 0 11-8 0"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:userOrders
      </a>
      <h3
        class="c3"
      >
        profile:researcherArea
      </h3>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M22 12h-6l-2 3h-4l-2-3H2"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M5.45 5.11L2 12v6a2 2 0 002 2h16a2 2 0 002-2v-6l-3.45-6.89A2 2 0 0016.76 4H7.24a2 2 0 00-1.79 1.11v0z"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:myTechnologies
      </a>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8 6H21"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M8 12H21"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M8 18H21"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M3 6H3.01"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M3 12H3.01"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M3 18H3.01"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:myServices
      </a>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 4h2a2 2 0 012 2v14a2 2 0 01-2 2H6a2 2 0 01-2-2V6a2 2 0 012-2h2"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M15 2H9a1 1 0 00-1 1v2a1 1 0 001 1h6a1 1 0 001-1V3a1 1 0 00-1-1z"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:orders
      </a>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 22c5.523 0 10-4.477 10-10S17.523 2 12 2 2 6.477 2 12s4.477 10 10 10z"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M9.09 9a3 3 0 015.83 1c0 2-3 3-3 3M12 17h.01"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:questions
      </a>
      <h3
        class="c3"
      >
        profile:reviewerArea
      </h3>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M20 21v-2a4 4 0 00-4-4H8a4 4 0 00-4 4v2M12 11a4 4 0 100-8 4 4 0 000 8z"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:profileCurate
      </a>
      <a
        class="c4"
      >
        <svg
          fill="none"
          height="24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11 19a8 8 0 100-16 8 8 0 000 16zM21 21l-4.35-4.35"
            stroke="hsla(0, 0%, 52%)"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
        profile:curateTechnologies
      </a>
      <a
        class="c4"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="hsla(0, 0%, 52%)"
          stroke-width="1.5"
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M536.1 273H488c-4.4 0-8 3.6-8 8v275.3c0 2.6 1.2 5 3.3 6.5l165.3 120.7c3.6 2.6 8.6 1.9 11.2-1.7l28.6-39c2.7-3.7 1.9-8.7-1.7-11.2L544.1 528.5V281c0-4.4-3.6-8-8-8zm219.8 75.2l156.8 38.3c5 1.2 9.9-2.6 9.9-7.7l.8-161.5c0-6.7-7.7-10.5-12.9-6.3L752.9 334.1a8 8 0 0 0 3 14.1zm167.7 301.1l-56.7-19.5a8 8 0 0 0-10.1 4.8c-1.9 5.1-3.9 10.1-6 15.1-17.8 42.1-43.3 80-75.9 112.5a353 353 0 0 1-112.5 75.9 352.18 352.18 0 0 1-137.7 27.8c-47.8 0-94.1-9.3-137.7-27.8a353 353 0 0 1-112.5-75.9c-32.5-32.5-58-70.4-75.9-112.5A353.44 353.44 0 0 1 171 512c0-47.8 9.3-94.2 27.8-137.8 17.8-42.1 43.3-80 75.9-112.5a353 353 0 0 1 112.5-75.9C430.6 167.3 477 158 524.8 158s94.1 9.3 137.7 27.8A353 353 0 0 1 775 261.7c10.2 10.3 19.8 21 28.6 32.3l59.8-46.8C784.7 146.6 662.2 81.9 524.6 82 285 82.1 92.6 276.7 95 516.4 97.4 751.9 288.9 942 524.8 942c185.5 0 343.5-117.6 403.7-282.3 1.5-4.2-.7-8.9-4.9-10.4z"
          />
        </svg>
        profile:revisionsHistory
      </a>
      <button
        class="c5"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M868 732h-70.3c-4.8 0-9.3 2.1-12.3 5.8-7 8.5-14.5 16.7-22.4 24.5a353.84 353.84 0 0 1-112.7 75.9A352.8 352.8 0 0 1 512.4 866c-47.9 0-94.3-9.4-137.9-27.8a353.84 353.84 0 0 1-112.7-75.9 353.28 353.28 0 0 1-76-112.5C167.3 606.2 158 559.9 158 512s9.4-94.2 27.8-137.8c17.8-42.1 43.4-80 76-112.5s70.5-58.1 112.7-75.9c43.6-18.4 90-27.8 137.9-27.8 47.9 0 94.3 9.3 137.9 27.8 42.2 17.8 80.1 43.4 112.7 75.9 7.9 7.9 15.3 16.1 22.4 24.5 3 3.7 7.6 5.8 12.3 5.8H868c6.3 0 10.2-7 6.7-12.3C798 160.5 663.8 81.6 511.3 82 271.7 82.6 79.6 277.1 82 516.4 84.4 751.9 276.2 942 512.4 942c152.1 0 285.7-78.8 362.3-197.7 3.4-5.3-.4-12.3-6.7-12.3zm88.9-226.3L815 393.7c-5.3-4.2-13-.4-13 6.3v76H488c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h314v76c0 6.7 7.8 10.5 13 6.3l141.9-112a8 8 0 0 0 0-12.6z"
          />
        </svg>
        profile:logout
      </button>
    </section>
    <section
      class="c6"
    >
      <h3
        class="c7"
      >
        profile:revisionsHistory
      </h3>
      <div
        class="c8"
      >
        <section
          class="c9"
        >
          <ol
            class="c10"
          >
            <div
              class="c11"
            >
              <div
                class="sort-by"
              >
                <div
                  class=" css-k377xg-container"
                >
                  <div
                    class=" css-yk16xz-control"
                  >
                    <div
                      class=" css-1qupxr1-ValueContainer"
                    >
                      <div
                        class=" css-1edv8nq-placeholder"
                      >
                        datagrid:sortByPlaceholder
                      </div>
                      <input
                        aria-autocomplete="list"
                        class="css-62g3xt-dummyInput"
                        id="react-select-bookmarks-ordering-select-input"
                        readonly=""
                        tabindex="0"
                        value=""
                      />
                    </div>
                    <div
                      class=" css-1hb7zxy-IndicatorsContainer"
                    >
                      <span
                        class=" css-1okebmr-indicatorSeparator"
                      />
                      <div
                        aria-hidden="true"
                        class=" css-tlfecz-indicatorContainer"
                      >
                        <svg
                          aria-hidden="true"
                          class="css-6q0nyr-Svg"
                          focusable="false"
                          height="20"
                          viewBox="0 0 20 20"
                          width="20"
                        >
                          <path
                            d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                          />
                        </svg>
                      </div>
                    </div>
                  </div>
                </div>
                
              </div>
              <div
                class="pagination"
              >
                <button
                  aria-label="First Page"
                  disabled=""
                  type="button"
                >
                  <svg
                    class="c12"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 1024 1024"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M272.9 512l265.4-339.1c4.1-5.2.4-12.9-6.3-12.9h-77.3c-4.9 0-9.6 2.3-12.6 6.1L186.8 492.3a31.99 31.99 0 0 0 0 39.5l255.3 326.1c3 3.9 7.7 6.1 12.6 6.1H532c6.7 0 10.4-7.7 6.3-12.9L272.9 512zm304 0l265.4-339.1c4.1-5.2.4-12.9-6.3-12.9h-77.3c-4.9 0-9.6 2.3-12.6 6.1L490.8 492.3a31.99 31.99 0 0 0 0 39.5l255.3 326.1c3 3.9 7.7 6.1 12.6 6.1H836c6.7 0 10.4-7.7 6.3-12.9L576.9 512z"
                    />
                  </svg>
                </button>
                <button
                  aria-label="Previous Page"
                  disabled=""
                  type="button"
                >
                  <svg
                    class="c13"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 1024 1024"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 0 0 0 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                    />
                  </svg>
                </button>
                <span>
                  1 - 3 datagrid:paginationLabel 3
                </span>
                <button
                  aria-label="Next Page"
                  disabled=""
                  type="button"
                >
                  <svg
                    class="c14"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 1024 1024"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M765.7 486.8L314.9 134.7A7.97 7.97 0 0 0 302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 0 0 0-50.4z"
                    />
                  </svg>
                </button>
                <button
                  aria-label="Last Page"
                  disabled=""
                  type="button"
                >
                  <svg
                    class="c15"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 1024 1024"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M533.2 492.3L277.9 166.1c-3-3.9-7.7-6.1-12.6-6.1H188c-6.7 0-10.4 7.7-6.3 12.9L447.1 512 181.7 851.1A7.98 7.98 0 0 0 188 864h77.3c4.9 0 9.6-2.3 12.6-6.1l255.3-326.1c9.1-11.7 9.1-27.9 0-39.5zm304 0L581.9 166.1c-3-3.9-7.7-6.1-12.6-6.1H492c-6.7 0-10.4 7.7-6.3 12.9L751.1 512 485.7 851.1A7.98 7.98 0 0 0 492 864h77.3c4.9 0 9.6-2.3 12.6-6.1l255.3-326.1c9.1-11.7 9.1-27.9 0-39.5z"
                    />
                  </svg>
                </button>
              </div>
            </div>
            <li
              class="c16"
            >
              <div
                class="c17"
                data-name="Título"
              >
                Título
              </div>
              <div
                class="c17"
                data-name="Status"
              >
                Status
              </div>
              <div
                class="c17"
                data-name="Última atualização"
              >
                Última atualização
              </div>
            </li>
            <li
              class="c18"
            >
              <div
                class="c17"
                data-name="Título"
              >
                Fake technology
              </div>
              <div
                class="c17"
                data-name="Status"
              >
                <span
                  class="c19"
                >
                  Aguardando análise
                </span>
              </div>
              <div
                class="c17"
                data-name="Última atualização"
              >
                19/10/2020
              </div>
            </li>
            <li
              class="c20"
            >
              <div
                class="c17"
                data-name="Título"
              >
                Fake technology
              </div>
              <div
                class="c17"
                data-name="Status"
              >
                <span
                  class="c21"
                >
                  Aguardando correção
                </span>
              </div>
              <div
                class="c17"
                data-name="Última atualização"
              >
                19/10/2020
              </div>
            </li>
            <li
              class="c18"
            >
              <div
                class="c17"
                data-name="Título"
              >
                Fake technology
              </div>
              <div
                class="c17"
                data-name="Status"
              >
                <span
                  class="c22"
                >
                  Correção efetuada
                </span>
              </div>
              <div
                class="c17"
                data-name="Última atualização"
              >
                19/10/2020
              </div>
            </li>
          </ol>
        </section>
      </div>
    </section>
  </div>
</div>
`;
